Theory assessment/Theory_assessment_Software_Engineering.md

```diff
@@ -0,0 +1,150 @@
+### 1. How does Object-oriented Programming differ from Process Oriented Programming? 
+A: Object-oriented programming is divided into the objects, but process-oriented programming is divided into smaller functions. OOP has bottom-up approach and data members can be public/private/default, which can also help hide the data and is more secure. POP has top-to-down approach and the data members are public, also data hiding is impossible. 
+Function overloading in OOP is possible, but not possible in POP. OOP is great for solving big problems, and it focuses on data, POP focuses on functions.
+
+### 2. What's polymorphism in OOP?
+A: Polymorphism in OOP means you can use same function for different purposes, used and reused many times. Example: you can overwrite something that has different outcome in parent class. Has positive influence to code, because can save time and makes easy to debug if needed.
```

Comment:
8/10 There's another aspect of Polymorphism which not described here. i.e. the method overloading. That is one of important concepts in polymorphism.